flash/core/serve/core.py

```diff
@@ -100,15 +100,13 @@ class GridModel:
     *  How to handle ``__init__`` args not recorded in hparams of ``pl.LightningModule``
     """
 
+    @_requires_extras("serve")
```

Comment:
do you really need all, even COCO?

Reply:
Not sure what you mean here, this function just requires serve. Serve tests import "all" extras so that we can test all servable tasks (which each need their own extras)